modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -20,13 +21,19 @@ type KeeperTestSuite struct {
 	chainA *ibctesting.TestChain
 	chainB *ibctesting.TestChain
 	chainC *ibctesting.TestChain
+
+	queryClient types.QueryClient
 }
 
 func (suite *KeeperTestSuite) SetupTest() {
 	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 3)
 	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(0))
 	suite.chainB = suite.coordinator.GetChain(ibctesting.GetChainID(1))
 	suite.chainC = suite.coordinator.GetChain(ibctesting.GetChainID(2))
+
+	queryHelper := baseapp.NewQueryServerTestHelper(suite.chainA.GetContext(), suite.chainA.GetSimApp().InterfaceRegistry())
+	types.RegisterQueryServer(queryHelper, suite.chainA.GetSimApp().ICAKeeper)
+	suite.queryClient = types.NewQueryClient(queryHelper)
```

Comment:
This looks like the legacy grpc testing, you should be able to use the query server without adding a field, see #143

Reply:
Ah, nice! I had just referenced what had been done in `apps/transfer`

Will include with other changes in a follow up PR.

Reply:
I took a look at this. For `apps/` I don't think we can't use `suite.chainA.QueryServer.` as the query server that exists on the `ibctesting.TestChain` is defined [here](https://github.com/cosmos/ibc-go/blob/main/modules/core/types/query.go) and only encapsulates core. 

The other option which removes the `queryClient` field is to invoke the `InterchainAccountAddress` func directly on the `ICAKeeper`. 

```
res, err := suite.chainA.GetSimApp().ICAKeeper.InterchainAccountAddress(ctx, req)
```

What do you think?

Reply:
Let's do the second option you outlined since the QueryServer is just the keeper [anyways](https://github.com/cosmos/ibc-go/blob/74182d8d45bca3504ed0190e9cdedb1a8ba45973/testing/chain.go#L108) :+1: